src/diffusers/pipelines/unclip/pipeline_unclip.py

```diff
@@ -0,0 +1,360 @@
+# TODO License
```

Comment:
@patrickvonplaten can you help here? I actually don't know how we license our code